base_delivery_carrier_label/models/stock_picking.py

```diff
@@ -147,15 +147,22 @@ def create(self, vals):
         vals = self._values_with_carrier_options(vals)
         return super().create(vals)
 
+    def _get_carrier_account_domain(self):
+        return [
+            ("delivery_type", "in", self.mapped("delivery_type")),
+            "|",
+            ("company_id", "=", False),
+            ("company_id", "in", self.mapped("company_id.id")),
+            "|",
+            ("carrier_ids", "=", False),
+            ("carrier_ids", "in", self.mapped("carrier_id").ids),
```

Comment:
```suggestion
            ("company_id", "in", ('False' + self.mapped("company_id").ids)),
            "|",
            ("carrier_ids", "in", (False + self.mapped("carrier_id").ids)),
```

Reply:
Your proposal does not seem to work.
domain `('x2many', 'in', [False])` does not seem to work.